mern-ecommerce/models/order.js

```diff
@@ -21,7 +21,7 @@ const OrderSchema = new mongoose.Schema(
     amount: { type: Number },
     address: String,
     status: {
-      type: String,
+      type: Boolean,
```

Comment:
The type of the 'status' field in the OrderSchema should be String, not Boolean, since it's an enum of string values.